concordium-std-derive/src/lib.rs

```diff
@@ -1048,6 +1060,37 @@ fn contract_function_optional_args_tokens(
     (setup_fn_args, fn_args)
 }
 
+#[cfg(feature = "build-schema")]
+fn contract_function_schema_tokens2(
+    event_option: Option<syn::LitStr>,
+    contract_name: String,
```

Comment:
It would probably be best if you took in a `rust_name`, as we do for `contract_function_schema`.

And name the function using that name. That will avoid that issue where you had to rename some contract names.